scripts/restore-test-data.js

```diff
@@ -0,0 +1,98 @@
+const https = require('https');
```

Comment:
This is impressive, but you're working way harder than you have to here by using promise chains (calling `then()` and `catch()` functions in chains etc). Using promise constructors and chains is not necessary anymore. You should switch this up to use `async/await` so your code can read as a simple linear series of steps. It's a lot easier to maintain.

There's a pretty good tutorial here:

https://javascript.info/async-await